Documentation/PackedAssetBuilds.md

```diff
@@ -1,64 +1,121 @@
+```shell
+C:\git> git clone https://github.com/o3de/o3de
+```
 
-### Test the profile build
+Navigate to `C:\git\o3de`.
+Create a build folder for non-monolithic build of the engine, for example `C:\git\o3de\build\non_mono`.
 
-* Open the game in editor
-    * load `NewStarBase` level
-    * Verify that game can launch and connect to local server from editor
-* Validate local game launcher can connect to local server
+> The location and the name of the build folder is optional.
+
+Configure the engine in a non-monolithic mode with o3de-multiplayersample project.
+
+```shell
+C:\git\o3de> cmake -S . -B build\non_mono -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
+```
+
+> `-DLY_MONOLITHIC_GAME=OFF` is the default value but for clarity it's specified here explicitly.
+
+Build the Editor. This will compile the project and necessary gems to produce the required game assets.
+
+```shell
+C:\git\o3de> cmake --build build\non_mono --target Editor --config profile
+```
+
+Run the Asset Processor with o3de-multiplayersample and let all the assets get processed.
+
+```shell
+C:\git\o3de> .\build\non_mono\bin\profile\AssetProcessor.exe --project-path C:\git\o3de-multiplayersample
+```
 
 ### Build AssetBuilder
 
-You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built.
+You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built to create game bundles (.pak files).
 
 For example:
 ```shell
-cmake --build build\windows --target AssetBundler --config profile -- /m /nologo
+C:\git\o3de> cmake --build build\non_mono --target AssetBundler --config profile
 ```
 
 ### Build monolithic game
 
 Build a second version of the executables as monolithic pak builds.
 
-``` shell
+> A separate build folder is required for building monolithic binaries, separate from the non-monolithic build folder.
+
+```shell
 # Create build files for a monolithic build that also disables all user/project registry settings overrides
-cmake -B build\windows_mono -S . -G "Visual Studio 16" -DLY_3RDPARTY_PATH=c:\your\path\to\3rdParty -DLY_MONOLITHIC_GAME=1 -DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0
+C:\git\o3de> cmake -S . -B build\mono -DLY_MONOLITHIC_GAME=1 -DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0 -DLY_PROJECTS="C:\git\o3de-multiplayersample"
 
 # Build the profile versions of all the executables
-cmake --build build\windows_mono --target MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher MultiplayerSample.UnifiedLauncher --config profile -- /m /nologo
+C:\git\o3de> cmake --build build\mono --target MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher MultiplayerSample.UnifiedLauncher --config profile
+```
+
+Profile monolithic game binaries will be located in `C:\git\o3de\build\mono\bin\profile`.
+Optionally, you can build monolithic release game binaries.
```

Comment:
I wonder if we should have a warning here about how if they build in release mode the build won't have access to the console (unless the cvar is enabled) and there may be fewer logs etc.